.github/workflows/build.yml

```diff
@@ -40,28 +62,29 @@ jobs:
   integration-test:
```

Comment:
Both integration and unit tests have the same structure. Is there a way to parameterize this?